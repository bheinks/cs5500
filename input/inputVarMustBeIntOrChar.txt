(* 
   Input variable must be of type integer or char
*)

program p;

var x : boolean;

begin
  read(x);
  write('!')
end.

