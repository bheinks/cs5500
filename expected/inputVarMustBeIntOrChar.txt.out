Line 10: Input variable must be of type integer or char
